src/model/generateRestHandlers.ts

```diff
@@ -72,12 +123,16 @@ export function generateRestHandlers<
         const cursor = req.url.searchParams.get('cursor')
         const rawSkip = req.url.searchParams.get('skip')
         const rawTake = req.url.searchParams.get('take')
```

Comment:
Brainstorming here: do you think it would make sense to introduce a `parseQueryParams` function that would:

- Parse pagination properties and convert them to numbers, as needed.
- Return all the _rest_ properties and treat them as `filters`

```ts
function parseQueryParams(params: URLSearchParams): ParsedQueryParams

interface ParsedQueryParams {
  [property: string]: string
  take?: number
  skip?: number
  cursor?: PrimaryKeyType
}
```

It feels like we access query parameters multiple times but in an inconsistent manner.

Reply:
When working on filtes I was thinking the same, have a single entry point for it. I think that it makes sense